components/ledger/pkg/storage/sqlstorage/ledger/accounts.go

```diff
@@ -0,0 +1,323 @@
+package ledger
+
+import (
+	"context"
+	"database/sql"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/formancehq/ledger/pkg/core"
+	"github.com/formancehq/ledger/pkg/ledger"
+	"github.com/formancehq/stack/libs/go-libs/api"
+	"github.com/pkg/errors"
+	"github.com/uptrace/bun"
+)
+
+const accountsTableName = "accounts"
+
+type Accounts struct {
+	bun.BaseModel `bun:"accounts,alias:accounts"`
+
+	Address  string         `bun:"address,type:varchar,unique,notnull"`
+	Metadata map[string]any `bun:"metadata,type:jsonb,default:'{}'"`
+}
+
+//------------------------------------------------------------------------------
```

Comment:
Sorry, this kind of separation is not part of our guidelines, either we discuss about a convention, or we remove this.

Reply:
No worries, I removed it !